src/Command/GenerateCommand.php

```diff
@@ -66,16 +43,24 @@ protected function execute(InputInterface $input, OutputInterface $output): void
                 }
             }
         } else {
-            $output->write('<info>If you want to create empty migration, use <fg=yellow>migrate/create</></info>');
+            $output->writeln('<info>If you want to create new empty migration, use <fg=yellow>migrate/create</></info>');
 
-            // if ($input->isInteractive() && $input instanceof StreamableInputInterface) {
-            //     $output->write('Would you like to create empty migration? (Y/n): ');
-            //     $answer = fgets($input->getStream() ?? STDIN);
-            //     if (in_array(strtolower(trim($answer)), ['yes', 'y'])) {
-            //         // create empty migration
-            //         $this->cycleHelper->generateEmptyMigration($this->migrator, $this->config);
-            //     }
-            // }
+            if ($input->isInteractive() && $input instanceof StreamableInputInterface) {
+                $output->write('Would you like to create empty migration right now? (Y/n): ');
```

Comment:
https://symfony.com/doc/5.0/components/console/helpers/questionhelper.html could be used, I think.